src/Symfony/Component/Mailer/Bridge/Mailgun/RemoteEvent/MailgunPayloadConverter.php

```diff
@@ -53,4 +53,19 @@ public function convert(array $payload): AbstractMailerEvent
 
         return $event;
     }
+
+    private function getReason(array $payload): string
+    {
+        if ('' !== $payload['delivery-status']['description']) {
```

Comment:
does the key already exist in the payload or do we need to use `?? ''` ?

Reply:
The key can exist, some errors return a description (suppress bounces for example), others the message from the email server.

description is more actionnable when given, message is the brute response so can sometimes be complex to understand but gives more insight than the simple bounce/suppress-bounce reason

Reply:
Oh sorry, I think I understood. From my experience, this key is always there, empty if no content provided.